### PR TITLE
Update Trillian to latest

### PR DESCRIPTION
**Pull request by AlCutter**
- Update Trillian to 4ef5996
- go mod tidy

### Checklist

- [ ] I have updated the [CHANGELOG](CHANGELOG.md).
  - Adjust the draft version number according to [semantic versioning](https://semver.org/) rules.
- [ ] I have updated [documentation](docs/) accordingly.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,23 +3,22 @@ module github.com/google/certificate-transparency-go
 go 1.16
 
 require (
-	bitbucket.org/creachadair/shell v0.0.6 // indirect
```
Comment (pphaneuf) [inline go.mod:6]: This is disappearing again, expected?

**Review by pphaneuf — COMMENTED** [review]
Did you do a `go generate ./...` as well?

**Comment by AlCutter** [comment]
> Did you do a `go generate ./...` as well?

Yup, no delta from checked-in files.

File: go.mod (hunk 1 above)
Comment (AlCutter) [reply go.mod:6]: well, "expected" :) 
(the CI build works as-is)

Comment (pphaneuf) [reply go.mod:6]: Hahaha! Well, as long as the CI is happy... 😃

**Review by pphaneuf — APPROVED** [review]

**Pull request merged**